packages/material-ui/src/Portal/Portal.js

```diff
@@ -50,14 +50,18 @@ Portal.propTypes = {
   /**
    * The children to render into the `container`.
    */
-  children: PropTypes.node.isRequired,
+  children: PropTypes.element.isRequired,
```

Comment:
Conflict, based on the comment, `node` is correct?

Reply:
Tricky. Looking at the code it needs to be a single element if `disablePortal` but `node` if we actually portal.

Without any context I would just remove `disablePortal`:
```diff
- <Portal disablePortal />
+ disablePortal ? React.Fragment : Portal
```

I lean towards widening the typescript definition to `node` and leave the rest to runtime checking with `React.Children.only` + `React.cloneElement`.

Reply:
Changed it to `node`